src/platform/osx.m

```diff
@@ -164,9 +164,11 @@ int platform_open_common_file_dialog(int type, utf8 *title, utf8 *filename, utf8
 		panel.allowedFileTypes = extensions;
 		panel.directoryURL = [NSURL fileURLWithPath:directory];
 		if ([panel runModal] == NSFileHandlingPanelCancelButton){
+			SDL_RaiseWindow(gWindow);
```

Comment:
If you did that then the return statement would also have to be outside along with a result variable that is set. Ultimately more code.